packages/mobile/src/home/NotificationBox.test.tsx

```diff
@@ -96,7 +112,39 @@ describe('NotificationBox', () => {
         <NotificationBox />
       </Provider>
     )
-    expect(getByText('incomingPaymentRequest')).toBeTruthy()
+    expect(getByText('incomingPaymentRequests')).toBeTruthy()
+  })
+
+  it('renders outgoing payment requests when they exist', () => {
+    const store = createMockStore({
+      ...storeDataNotificationsDisabled,
+      account: {
+        ...storeDataNotificationsDisabled.account,
+        outgoingPaymentRequests: mockPaymentRequests,
+      },
+    })
+    const { getByText } = render(
+      <Provider store={store}>
+        <NotificationBox />
+      </Provider>
+    )
+    expect(getByText('outgoingPaymentRequests')).toBeTruthy()
+  })
+
+  it('renders outgoing payment request when they exist', () => {
```

Comment:
How does this differ from the test above?

Reply:
Single payment request vs multiple

Reply:
Can we clarify the test description? I think the only diff is one character ;)